client/assets/script.js

```diff
@@ -0,0 +1,16 @@
+// DOM manipulation
+
+const form = document.getElementsByTagName('form')
```

Comment:
This is generating an error in the console log as it says form is not defined. I think it has something to do with using getElementsByTagName. The following code works or at least gets rid of the error:

const form = document.querySelector('form')
form.addEventListener('submit', getAllResults)

Reply:
Thank you Karl, I will try change that